airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java

```diff
@@ -188,6 +189,7 @@
             "\\\\ \\\\\n" +
             "To change or remove this message, log in with administrator rights and go to <a href='settings.view'>Settings</a> > <a href='generalSettings.view'>General</a>.";
     private static final String DEFAULT_LOGIN_MESSAGE = null;
+    private static final int DEFAULT_SESSION_TIMEOUT_MINUTES = 60;
```

Comment:
Maybe a String or Duration?

org.springframework.boot.web.servlet.server.Session details the timeout field as a Duration

default is 30m it seems